src/style/theme.scss

```diff
@@ -60,6 +61,62 @@ $iui-icons-color-positive--light: $iui-color-foreground-positive--light;
 $iui-icons-color-negative--light: $iui-color-foreground-negative--light;
 $iui-icons-color-warning--light: $iui-color-foreground-warning--light;
 
+// LIGHT HIGH CONTRAST THEME //
+// Light high contrast theme > Background
+$iui-color-background-1--light-hc: #FFF;
+$iui-color-background-2--light-hc: #F2F2F2;
+$iui-color-background-3--light-hc: #E5E5E5;
+$iui-color-background-4--light-hc: #D9D9D9;
+$iui-color-background-5--light-hc: #CCC;
+$iui-color-background-primary--light-hc: #0071B8;
```

Comment:
Should we test colors on all backgrounds? 
Warning orange fails on background-3
![image](https://user-images.githubusercontent.com/83585998/156363554-200410b6-f953-447b-89d1-27a558169a94.png)

Reply:
I made a test page to more easily check our scores.  The primary & warning colors were failing on background 3.  I've adjusted the colors ever so slightly to ensure we are passing.  Background 4 and 5 are mostly used for borders, so I think we can ignore scores against those backgrounds.